### ISSUE TITLE
Add Tensor Manipulation ops to TTNN

### ISSUE THREAD

**Issue by Aswinmcw** [tenstorrent-metal/tt-metal#5326]
**Is your feature request related to a problem? Please describe.**
We need a new operation to perform tensor manipulation operations similar to what was implemented in torch.
ex. [zeros](https://pytorch.org/docs/stable/generated/torch.zeros.html) , [full_like](https://pytorch.org/docs/stable/generated/torch.full_like.html)
Following are the ops needed
normalize_global, normalize_hw, ones, zeros, zeros_like, full, full_like, mean_hw, maeloss, mseloss, std_hw, softplus, var_hw

**Describe the solution you'd like**
This will perform tensor manip operation to the given input tensor and return one output tensor.

**Describe alternatives you've considered**
It's possible that ttnn tensor could be converted to a pytorch and then do tensor manip operations with pytorch.  Then the result would be returned by using a ttnn.from_torch operation.

**Additional context**
The reference implementation in this case is:
```
import torch

out=torch.zeros(2,3)
print(out)
```

**Comment by Aswinmcw** [comment]
Merged to main
https://github.com/tenstorrent-metal/tt-metal/pull/5410
